src/pages/SignUp/SignUp.tsx

```diff
@@ -1,47 +1,98 @@
 import React from 'react'
 import { useState, useEffect } from 'react'
-import { useLocation } from 'react-router-dom'
+import { useLocation, useNavigate } from 'react-router-dom'
 import Button from '@components/Button'
 import Input from '@components/Input'
+import useForm from '@hooks/useForm'
+import { useGetDuplicateNickname } from '@react-query/hooks/useNickname'
 
-type SingUpUserInfo = {
-  tempId: null | string
-  nickname: string
-}
+type PropertyType = 'default' | 'success' | 'error'
 
 export default function SignUp() {
   const location = useLocation()
-  const [userInfo, setUserInfo] = useState<SingUpUserInfo>({
-    tempId: null,
-    nickname: '',
+  const [nickname, setNickname] = useState('')
+  const [checkedNickname, setCheckedNickname] = useState(false)
+  const [property, setProperty] = useState<PropertyType>('default')
```

Comment:
```suggestion
  const [property, setProperty] = useState<PropertyType>('default')
```
boolean값을 반환하는 변수이름은 아래에서 그러셨던덕처럼 is를 붙이는게 더 이해가 편할것 같아용

Reply:
혹시 checkedNickname을 말하시는걸까요? <- 이부분 반영하겠습니다!